metacat-main/src/main/java/com/netflix/metacat/main/api/ExceptionMapper.java

```diff
@@ -66,18 +67,26 @@ public void handleMetacatException(
         } else if (e instanceof MetacatNotFoundException) {
             status = HttpStatus.NOT_FOUND.value();
         } else if (e instanceof MetacatNotSupportedException) {
+            logErrorLevel = true;
             status = HttpStatus.NOT_IMPLEMENTED.value();
         } else if (e instanceof MetacatUserMetadataException) {
             // TODO: This makes no sense
             status = HttpStatus.SEE_OTHER.value();
         } else if (e instanceof MetacatTooManyRequestsException) {
+            logErrorLevel = true;
             status = HttpStatus.TOO_MANY_REQUESTS.value();
         } else if (e instanceof MetacatUnAuthorizedException) {
+            logErrorLevel = true;
             status = HttpStatus.FORBIDDEN.value();
         } else {
+            logErrorLevel = true;
             status = HttpStatus.INTERNAL_SERVER_ERROR.value();
         }
-        log.error(e.getLocalizedMessage(), e);
+        if (logErrorLevel) {
+            log.error(e.getLocalizedMessage(), e);
+        } else {
+            log.warn(e.getLocalizedMessage(), e);
```

Comment:
We can also probably omit the stack trace for this one.

Reply:
Would like to keep logging stack trace for now, this PR is mainly focused on separating the two types so as to be more easily searchable and distinguishable in something like radar.